plugin/customcertificate/resources/css/form.css

```diff
@@ -15,4 +15,16 @@
     margin: 0 10px 0 10px;
 }
 input:disabled { background: #eee; }
-label { font-size: 13px; }
\ No newline at end of file
+label { font-size: 13px; }
+
+.list-tags{
+    display: block;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+.list-tags li{
+    display: inline-block;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/app/chamilo/chamilo-lms/pullRequest?prid=4644216)